lib/commons/dom/get-target-size.js

```diff
@@ -0,0 +1,70 @@
+import findNearbyElms from './find-nearby-elms';
+import { splitRects, hasVisualOverlap } from '../math';
+import memoize from '../../core/utils/memoize';
+
+const roundingMargin = 0.05;
+
+export default memoize(getTargetSize);
+
+/**
+ * Compute the target size of an element.
+ * @see https://www.w3.org/TR/WCAG22/#dfn-targets
+ */
+function getTargetSize(vNode, minSize) {
+  const nodeRect = vNode.boundingClientRect;
+  const overlappingVNodes = findNearbyElms(vNode).filter(vNeighbor => {
+    return (
+      vNeighbor.getComputedStylePropertyValue('pointer-events') !== 'none' &&
+      hasVisualOverlap(vNode, vNeighbor)
+    );
+  });
+
+  if (!overlappingVNodes.length) {
+    return {
+      rect: nodeRect
+    };
```

Comment:
I left this as an object because we'll probably need to add more information to this return later, such as the bounding rect of the target and the largests target size rect.